src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -567,9 +572,9 @@ internal String ToString(TraceFormat traceFormat)
                         String typeName = "<UnknownType>";
                         if (pi[j].ParameterType != null)
                             typeName = pi[j].ParameterType.Name;
-                        sb.Append(typeName + " " + pi[j].Name);             
+                        sb.Append(typeName).Append(' ').Append(pi[j].Name);             
```

Comment:
BTW, `Append` is used "fluently" elsewhere in mscorlib in a few places, but I changed the calls to be on their own lines for consistency here.